### PR TITLE
Feat/4: Test form layout

### PR DESCRIPTION
**Pull request by harshbrahmxatriya**
This is an initial commit to test the UI/Frontend, so form is will be rendered on the page, team can review.
How to test:
Go to URL: https://deploy-preview-39--wehelpvip.netlify.app/demo-get-involved
Part of feat #4

### CODE REVIEW

File: pages/cars.vue [hunk 1]
```diff
@@ -0,0 +1,52 @@
+<template>
+  <Suspense>
+    <template #default>
+      <div
+        v-for="car in list"
```
Comment (razbakov) [inline pages/cars.vue:5]: task about cars should not be part of this PR

File: pages/demo-get-involved.vue [hunk 2]
```diff
@@ -0,0 +1,237 @@
...
+          >
+            Save
+          </button>
+        </div>
+      </div>
+    </div>
+  </form>
+</template>
+<style>
+.container {
```
Comment (razbakov) [inline pages/demo-get-involved.vue:233]: don't use css, use tailwind

**Review by razbakov — CHANGES_REQUESTED** [review]

File: pages/demo-get-involved.vue [hunk 3]
```diff
@@ -0,0 +1,237 @@
+<template>
```
Comment (razbakov) [inline pages/demo-get-involved.vue:1]: Why `demo-get-involved`?

File: pages/demo-get-involved.vue [hunk 4]
```diff
@@ -0,0 +1,237 @@
...
+          </div>
+
+          <div
+            class="sm:grid sm:grid-cols-3 sm:gap-4 sm:items-start sm:border-t sm:border-gray-200 sm:pt-5"
+          >
+            <label
+              for="region"
+              class="block text-sm font-medium text-gray-700 sm:mt-px sm:pt-2"
+            >
+              State / Province
```
Comment (razbakov) [inline pages/demo-get-involved.vue:176]: Please see the original form and use same fields.

**Review by razbakov — CHANGES_REQUESTED** [review]

File: pages/demo-get-involved.vue (hunk 3 above)
Comment (harshbrahmxatriya) [reply pages/demo-get-involved.vue:1]: Because it is a demo/development version of the already made get-involved page

Comment (razbakov) [reply pages/demo-get-involved.vue:1]: in your demo you should replace the existing page

File: pages/demo-get-involved.vue (hunk 4 above)
Comment (harshbrahmxatriya) [reply pages/demo-get-involved.vue:176]: So no other fields like city zipcode country?

File: pages/cars.vue (hunk 1 above)
Comment (harshbrahmxatriya) [reply pages/cars.vue:5]: Don't know why it showed up again

Comment (razbakov) [reply pages/cars.vue:5]: because commits are in this branch, you need to remove commits that are unrelated

File: pages/demo-get-involved.vue (hunk 4 above)
Comment (razbakov) [reply pages/demo-get-involved.vue:176]: same fields as in original form

**Comment by razbakov** [comment]
Why did you delete the branch? You can change commits in the branch, you can use force push, cherry-pick, rebase.

**Comment by harshbrahmxatriya** [comment]
I rebased it but the commits were there, so just went ahead and added it again
